resources-domain/resources-service/package.json

```diff
@@ -42,6 +42,8 @@
     "db:undo-last": "sequelize db:migrate:undo",
     "es:create-index:development": "cross-env NODE_ENV=development tsx ./scripts/elasticsearch/create-index.ts",
     "es:delete-index:development": "cross-env NODE_ENV=development tsx ./scripts/elasticsearch/delete-index.ts",
+    "es:create-index:staging": "cross-env NODE_ENV=staging tsx ./scripts/elasticsearch/create-index.ts",
+    "es:delete-index:staging": "cross-env NODE_ENV=staging tsx ./scripts/elasticsearch/delete-index.ts",
```

Comment:
What's the reason to not use auto-create on this indices?

Reply:
Will that work for resources as well as HRM?